src/main/java/zmaster587/advancedRocketry/inventory/GuiOreMappingSatellite.java

```diff
@@ -55,10 +56,8 @@ public GuiOreMappingSatellite(SatelliteOreMapping satellite,EntityPlayer invento
 		playerPosZ = zCenter = (int) inventoryPlayer.posZ;
 
 		//Max zoom is 128
-		if(satellite != null) {
```

Comment:
Why did this go away? The satellite can still be null if it's unlaunched.

Reply:
Hmm.... I swear I've had NPEs without that so... yeah.

Reply:
It looks like the ContainerOreMappingSattelite would actually throw an NPE currently if it were passed a null SatteliteOreMapping, so a null would never make its way to the zoom calculations.

Reply:
Ah....